### ISSUE TITLE
Executing scss-lint directly finds issues but grunt scsslint does not.

### ISSUE THREAD

**Issue by Vpr99** [ahmednuaman/grunt-scss-lint#59]
I'm running the task on OSX using the default ruby installation (/usr/bin/ruby) and v0.26.2. When I try to run scss-lint directly from the the command line, it finds errors. However, when I try to run it through the Grunt task, no errors are found.

```
grunt scsslint
Running "scsslint:allFiles" (scsslint) task
Running scss-lint on allFiles
>> 1 file is lint free
Results have been written to: scss-lint-report.xml
Done, without errors.

scss-lint lib/scss/
lib/scss/_blog.scss:2 [W] Indentation: Line should be indented with spaces, not tabs
lib/scss/_blog.scss:2 [W] Shorthand: Shorthand form for property `margin` should be written more concisely as `30px 0 0` instead of `30px 0 0 0`
lib/scss/_blog.scss:3 [W] EmptyLineBetweenBlocks: Rule declaration should be preceded by an empty line
lib/scss/_blog.scss:3 [W] Indentation: Line should be indented with spaces, not tabs
```

My configuration for the Grunt task is as follows:

```
  allFiles: [
    'lib/scss/*.scss',
  ],
  options: {
    colorizeOutput: true
  },
```

Thanks in advance!
